src/panel_material_ui/template/Page.jsx

```diff
@@ -56,8 +56,16 @@ export function render({model, view}) {
   const [dark_theme, setDarkTheme] = model.useState("dark_theme")
   const [logo] = model.useState("logo")
   const [open, setOpen] = model.useState("sidebar_open")
-  const [sidebar_width] = model.useState("sidebar_width")
+  const [sidebar_width, setSidebarWidth] = model.useState("sidebar_width")
   const [theme_toggle] = model.useState("theme_toggle")
+
+  // Draggable sidebar state and constants
+  const [isDragging, setIsDragging] = React.useState(false)
+  const [dragStartX, setDragStartX] = React.useState(0)
+  const [dragStartWidth, setDragStartWidth] = React.useState(0)
+  const MIN_SIDEBAR_WIDTH = 180
+  const MAX_SIDEBAR_WIDTH = 600
+  const DEFAULT_SIDEBAR_WIDTH = 240
```

Comment:
previously we've had a default sidebar widget around 300px. Implications of making this smaller should be tested.